config/DS918+/7.0-41890/ramdisk-003-post-init-script.patch

```diff
@@ -0,0 +1,30 @@
+--- usr/sbin/init.post
```

Comment:
Shouldn't this one be a symlink as well?

Reply:
same as by the impl patch. Some changes in DSM7.